CHANGES

```diff
@@ -81,6 +97,9 @@ Standard Library
 
 Plugins
 
+- The Ssreflect plugin is now distributed with Coq. Its documentation has
+  been integrated as a chapter of the reference manual. This chapter is
+  still imperfect so feedback is welcome.
```

Comment:
I used "world in progress". Preliminary sounds like préliminaire and this word doesn't sound right in French.